ingest/scripts/diff-ndjson

```diff
@@ -105,9 +105,16 @@ def compare_records(truth_record, query_record):
 
 def remove_sequence(record):
     """Remove the sequence string (ATCGs) to save memory"""
-    for s in record.get('sequences', []):
-        if 'sequence' in s:
-            del s['sequence']
+    sequences = record.get('sequences')
+    if isinstance(sequences, list):
+        for s in sequences:
+            if 'sequence' in s:
+                del s['sequence']
```

Comment:
from https://github.com/nextstrain/seasonal-flu/pull/274#pullrequestreview-3565191415:

> Question: record['sequences'] will only be a dict from this point on, but allowing diff-ndjson to handle lists is for backwards compat? If so - thank you!!!
 
Yes, I kept the list support in place for backwards comparison. This let me compare a small dataset new NDJSON against the old NDJSON to make sure the only thing that's changed was the type of the `sequences` field.